src/apps/competitions/tasks.py

```diff
@@ -338,6 +361,17 @@ def unpack_competition(competition_dataset_pk):
                 "solutions": {},
             }
 
+            # Get Queue by vhost/uuid. If instance not returned, or we don't have access don't set it!
+            if competition_yaml.get('queue'):
+                try:
+                    queue = Queue.objects.get(vhost=competition_yaml.get('queue'))
+                    if not queue.is_public:
+                        if queue.owner != creator or creator.username not in queue.organizers.all().values_list('username'):
```

Comment:
`values_list('username', flat=True)` is probably more what ya want, otherwise:
![image](https://user-images.githubusercontent.com/37089863/68520675-73b27000-024e-11ea-8b40-a86cba425416.png)